mitogen/master.py

```diff
@@ -148,7 +148,11 @@ def is_stdlib_name(modname):
         return False
 
     # six installs crap with no __file__
-    modpath = os.path.abspath(getattr(module, '__file__', ''))
+    path = getattr(module, '__file__', '')
+    if path is None:
```

Comment:
Adding the package might be tricky, since it seems to be a part of Python distribution, though I'm not 100% sure.
The package was `backports`.

Will try to reproduce on different environments

Reply:
Ah, I see, thanks for clarification :+1: